workshop/Lab3/README.md

```diff
@@ -0,0 +1,285 @@
+# Lab 3: Scale and update apps natively, building multi-tier applications.
+
+In this lab you'll learn how to deploy the same guestbook application we deployed
+in the previous labs, however, instead of using the `kubectl` command line helper
+functions we'll be deploying the application using configuration files. The configuration
+file mechanism allows you to have more fine-grained control over all of resources
+being created within the Kubernetes cluster.
+
+Before we work with the application we need to clone a github repo from which we can get 
+all our configuration files.
+
+``` 
+	$git clone git@github.com:IBM/guestbook.git
+```
+
+# 1. Scale apps natively
+
+Kubernetes can deploy an individual pod to run an applications but when you need to
+scale it to handle a large number of requests a `Deployment` is the resource you
+want to use. A Deployment manages a collection of similar pods (images) and
+based on the number of copies ("replicas") you ask for, the Deployment will try to
+ensure that you always have exactly that many copies of your pod running.
+
+Every Kubernetes object includes two nested object fields that govern the object’s configuration: the object spec and the object status. Object spec defines the desired state, and object status field contains kubernetes system provided information about the actual state of the resource. As described before Kubernetes system will make reconciliation happend between desired and actual state.
+Every Object that we create, we should provide the apiVersion you are suing to create the object, kind of the object you are creating and the metadata about the object such as a name, set of labels and optionally namespace that this object should belong. 
```

Comment:
s/suing/using/

Reply:
you really should put config file keywords in `back-ticks` so they stand out and don't run the risk of being read a normal English words.

Reply:
done